src/components/ebay-infotip/browser.json

```diff
@@ -1,6 +1,6 @@
 {
     "dependencies": [
         "../ebay-icon",
-        "require: ./index.js"
+        "require: ./index.marko"
```

Comment:
Neither of these deps should be needed. (Should be able to remove this browser.json).